joomla/joomla-cms
# [5.2] Error handling on extension update

Pull Request for Issue #43319  .

### Summary of Changes

Catch errors from the extension being updated and abort update with a sensible message.

### Testing Instructions

(a) Installed this Plugin.
[plg_updatetest1.zip](https://github.com/joomla/joomla-cms/files/15048374/plg_updatetest1.zip)

(b) **Enable the plugin.**

(c) Go to Joomla updater and check for updates.

(d) Then select the plugin installed above PLG_HIKASHOPSHIPPING_BFEXTRAFEATURES and update it.

### Actual result BEFORE applying this Pull Request

![Error 0](https://github.com/joomla/joomla-cms/assets/3941269/d6da8951-4325-49a0-a4f0-1abea52a4cf0)

### Expected result AFTER applying this Pull Request

![Error 1](https://github.com/joomla/joomla-cms/assets/3941269/e939d385-6f76-48d7-8c0d-60f25467f1da)


### Link to documentations
Please select:
